(DOCS-2753) Remove instances of 'easy' and 'easily' (#12469)
DataDog/documentation: content/en/real_user_monitoring/guide/send-rum-custom-actions.md

## Create facets and measures on your new attributes
Once you have deployed the code that creates your custom actions, you will start seeing actions appear in the [RUM Explorer][3], in the **Actions** tab.

> To filter on your new custom Actions, use the `Action Target Name` attribute as follow: `@action.target.name:<ACTION_NAME>`. In the example, we use the following filter: `@action.target.name:checkout`

Reviewer: ```suggestion
To filter on your new custom Actions, use the `Action Target Name` attribute as follow: `@action.target.name:<ACTION_NAME>`. The example uses the following filter: `@action.target.name:checkout`
```
